FreeRTOS-Plus/Demo/AWS/Device_Defender_Windows_Simulator/Device_Defender_Demo/demo_config.h

```diff
@@ -0,0 +1,301 @@
+ * then a UDP port after the network has connected. */
+extern void vLoggingPrintf( const char * pcFormatString,
+                            ... );
+
+/* Map the SdkLog macro to the logging function to enable logging
+ * on Windows simulator. */
+#ifndef SdkLog
+    #define SdkLog( message )    vLoggingPrintf message
+#endif
+
+#include "logging_stack.h"
+
+
+/************ End of logging configuration ****************/
+
+/**
+ * @brief The Thing resource registered on your AWS IoT account to use in the demo.
+ * A Thing resource is required to communicate with the AWS IoT Device Shadow service.
+ *
+ * @note The Things associated with your AWS account can be found in the
+ * AWS IoT console under Manage/Things, or using the ListThings REST API (that can
+ * be called with the AWS CLI command line tool).
+ *
+ * #define democonfigTHING_NAME    "...insert here..."
+ */
+
+#ifndef democonfigCLIENT_IDENTIFIER
+
+/**
+ * @brief The MQTT client identifier used in this example.  Each client identifier
+ * must be unique so edit as required to ensure no two clients connecting to the
+ * same broker use the same client identifier.
+ *
+ * @note Appending __TIME__ to the client id string will reduce the possibility of a
+ * client id collision in the broker. Note that the appended time is the compilation
+ * time. This client id can cause collision, if more than one instance of the same
+ * binary is used at the same time to connect to the broker.
+ */
+    #define democonfigCLIENT_IDENTIFIER    "testClient"__TIME__
+#endif
+
+/**
+ * @brief The AWS IoT broker endpoint to connect to in the demo.
+ *
+ * @note Your AWS IoT Core endpoint can be found in the AWS IoT console under
+ * Settings/Custom Endpoint, or using the DescribeEndpoint REST API (that can
+ * be called with AWS CLI command line tool).
+ *
+ * #define democonfigMQTT_BROKER_ENDPOINT    "...insert here..."
+ */
+
+/**
+ * @brief The port to use for the demo.
+ *
+ * In general, port 8883 is for secured MQTT connections.
+ *
+ * @note Port 443 requires use of the ALPN TLS extension with the ALPN protocol
+ * name. Using ALPN with this demo would require additional changes, including
+ * setting the `pAlpnProtos` member of the `NetworkCredentials_t` struct before
+ * forming the TLS connection. When using port 8883, ALPN is not required.
+ *
+ * #define democonfigMQTT_BROKER_PORT    ( insert here. )
+ */
+
+/**
+ * @brief AWS root CA certificate.
+ *
+ * This certificate is used to identify the AWS IoT server and is publicly available.
+ * Refer to the link below.
+ * https://www.amazontrust.com/repository/AmazonRootCA1.pem
+ *
+ * @note This certificate should be PEM-encoded.
+ *
+ * Must include the PEM header and footer:
+ * "-----BEGIN CERTIFICATE-----\n"\
+ * "...base64 data...\n"\
+ * "-----END CERTIFICATE-----\n"
+ *
+ */
+#define democonfigROOT_CA_PEM                                            \
+    "-----BEGIN CERTIFICATE-----\n"                                      \
```

Comment:
We should not be checking in certificates

Reply:
Removed. Perhaps should be removed from shadow demo as well?